apis/io-engine/protobuf/v1/pool.proto

```diff
@@ -59,23 +60,20 @@ message CreatePoolRequest {
 
 // Pool metadata arguments.
 message PoolMetadataArgs {
-  // This parameter controls the initial metadata reservation for a storage pool.
-  // This parameter sets a ratio of the desired metadata reservation amount
-  // to the default reservation for the given pool type.
+  // Deprecated after the addition of max_expansion.
+  optional float md_resv_ratio = 1;
```

Comment:
You could also add a `PoolMetadataArgsV2` btw, and use `GrowPoolRequestV2`

Reply:
GrowPoolRequest doesn't have any changes. So i guess we can reuse that in GrowPoolV2.

If we add a new message `PoolMetadataArgsV2`, the existing `PoolMetadataArgs` will still be there in `CreatePoolRequest`. Doesn't add much value i think.